std/range/package.d

```diff
@@ -7219,6 +7219,17 @@ if (isForwardRange!RangeOfRanges &&
     this(RangeOfRanges input)
     {
         this._input = input;
+        static if (opt == TransverseOptions.enforceNotJagged)
+        {
+            import std.exception : enforce;
+
+            if (empty) return;
+            immutable commonLength = _input.front.length;
+            foreach (e; _input)
```

Comment:
`_input.front`'s length is unnecessarily enforced to be equal to itself.

Reply:
I just copied the code from transverse. If you've got an idea how to remove this redundancy you should do this at transverse as well.